semgrep/tests/conftest.py

```diff
@@ -74,6 +75,8 @@ def _run_semgrep(
     :param output_format: which format to use
     :param stderr: whether to merge stderr into the returned string
     """
+    possibly_notify_user()
```

Comment:
Having this called in the test level means if we remove it from the actual code path of semgrep we wont actually catch that it was removed no?